decidim-core/app/assets/javascripts/decidim/widget.js

```diff
@@ -0,0 +1,19 @@
+function getDocHeight(doc) {
```

Comment:
We should use es6 like replacing these `var` with `const` or `let` whenever it's possible. You are also exposing both variables as globals. It would be nice if you include them in the global object `Decidim`.